src/glint.gleam

```diff
@@ -855,13 +875,34 @@ fn flags_help_to_string(help: List(FlagHelp), config: Config) -> String {
 /// generate the help text for a flag without a description
 ///
 fn flag_help_to_string(help: FlagHelp) -> String {
-  prefix <> help.meta.name <> "=<" <> help.type_ <> ">"
+  prefix
+  <> help.meta.name
+  <> case help.type_ {
+    "" -> ""
+    _ -> "=<" <> help.type_ <> ">"
+  }
 }
 
 /// generate the help text for a flag with a description
 ///
-fn flag_help_to_string_with_description(help: FlagHelp) -> String {
-  flag_help_to_string(help) <> "\t\t" <> help.meta.description
+fn flag_help_to_string_with_description(
+  help: FlagHelp,
+  longest_flag_length: Int,
+) -> String {
+  {
+    help
+    |> flag_help_to_string
+    |> string.pad_right(longest_flag_length, " ")
+  }
+  <> {
+    let description_width =
+      { max_output_width - longest_flag_length - tab_width }
+      |> int.max(min_column_width)
+
+    help.meta.description
+    |> utils.wordwrap(description_width)
+    |> string.join("\n\t" <> string.repeat(" ", longest_flag_length))
+  }
```

Comment:
🔧 considering that the description expression is the majority of the body of this function, we can move it up and bind the description to its own variable to avoid nesting

```suggestion
  let name =
    help
    |> flag_help_to_string
    |> string.pad_right(longest_flag_length, " ")
 
  let description_width =
    { max_output_width - longest_flag_length - tab_width }
    |> int.max(min_column_width)

  let description =
    help.meta.description
    |> utils.wordwrap(description_width)
    |> string.join("\n\t" <> string.repeat(" ", longest_flag_length))

name <> description
```